cmd/list_services.go

```diff
@@ -12,16 +13,22 @@ var listServicesCmd = &cobra.Command{
 	Short: "list services on PagerDuty",
 	Long:  "Get the list of services configured in PagerDuty",
 	Args:  cobra.ExactArgs(1),
-	RunE:  listServices,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		pd := &pagerDutyClient{client: api.NewPagerDutyAPIClient(Config.PdAuthToken)}
+		err := pd.listServices(args[0])
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
```suggestion
		return pd.listServices(args[0])
```
similar remark to the one I posted above
if we don't have any special logic for handling error why not return right away?

Reply:
fixed